src/Standards/Generic/Tests/Arrays/DisallowLongArraySyntaxUnitTest.php

```diff
@@ -26,12 +26,6 @@ protected function getTestFiles($testFileBase)
     {
         $testFiles = [$testFileBase.'1.inc'];
 
-        // HHVM doesn't tokenize any of the file after a git
```

Comment:
`if (defined('HHVM_VERSION') === false) {` === not HHVM, so this whole method can be removed.

Reply:
Oops "only run this check on non-HHVM versions" test needs to be added back, but the check can be removed

Reply:
If the `getTestFiles()` method is not there, PHPCS automatically will test all files which start with `SniffNameUnitTest....`, so if there is no logic needed to determine which files to test - i.e. what we're left with now -, the whole method can be removed.

Reply:
Oh, I see. I understand now what you are saying. I'll make that improvement in the morning.

Reply:
fixed